openfermioncirq/gates/common_gates.py

```diff
@@ -40,23 +39,22 @@ class FSwapPowGate(cirq.EigenGate,
     `ofc.FSWAP` is an instance of this gate at exponent=1. It swaps adjacent
     fermionic modes under the Jordan-Wigner Transform.
     """
+
     def num_qubits(self):
         return 2
 
     def _eigen_components(self):
         return [
-            (0, np.array([[1, 0,   0,   0],
-                          [0, 0.5, 0.5, 0],
-                          [0, 0.5, 0.5, 0],
-                          [0, 0,   0,   0]])),
-            (1, np.array([[0,  0,    0,   0],
-                          [0,  0.5, -0.5, 0],
-                          [0, -0.5,  0.5, 0],
-                          [0,  0,    0,   1]])),
+            (0,
+             np.array([[1, 0, 0, 0], [0, 0.5, 0.5, 0], [0, 0.5, 0.5, 0],
+                       [0, 0, 0, 0]])),
+            (1,
+             np.array([[0, 0, 0, 0], [0, 0.5, -0.5, 0], [0, -0.5, 0.5, 0],
+                       [0, 0, 0, 1]])),
```

Comment:
Optional nit: Previous formatting is clearly more readable. Consider displaying yapf for these four lines. [Example](https://github.com/quantumlib/Cirq/blob/9065d4e86720bcd2d68eff808663ee4f2106eeee/cirq/ops/swap_gates_test.py#L74).

Reply:
Done.